lms/djangoapps/discussion/rest_api/views.py

```diff
@@ -255,6 +255,14 @@ class ThreadViewSet(DeveloperErrorViewMixin, ViewSet):
 
         * read (optional): A boolean to mark thread as read
 
+        * edit_reason_code (optional): A string containing the reason
+        for editing the thread's body.
+
+        * closed (optional, privileged): A boolean to mark thread as closed.
```

Comment:
I've added documentation to `closed`, which already exists, but wasn't documented here.